content/en/docs/reference/glossary/deviceclass.md

```diff
@@ -0,0 +1,23 @@
+---
+title: DeviceClass
+id: deviceclass
+date: 2025-05-26
+short_description: >
+  Describes a category of devices in the cluster. Users can claim specific
+  devices in a DeviceClass.
+
+tags:
+- extension
+- core
+---
+ A DeviceClass describes a category of
+{{< glossary_tooltip text="devices" term_id="device" >}} in the cluster that can
+be used with
+[dynamic resource allocation (DRA)](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/).
+
+<!-- more -->
+
+Administrators or device owners use DeviceClasses to define what devices can be
+claimed and how to select specific attributes in a claim. Devices are claimed by
```

Comment:
I'm not sure a primary purpose of DeviceClasses is to "define how to select specific attributes in a claim." DeviceClasses are one place you could look for examples of CEL expressions, but the expressions used in ResourceClaims will almost always be different from DeviceClasses' expressions since requests already have to define a `deviceClassName`.